### DISCUSSION TITLE
download method

### DISCUSSION THREAD

**Discussion by veerman72** [hrynko/vue-pdf-embed#71] [Ideas]
Is there a possibility to create a public method (like the print() method) for direct downloading the rendered pdf. 

````
methods: {
    download () {
      this.$refs.myPdfComponent.download()
    },
}
````

I make use of a source like ``data:application/pdf;base64,<BASE64_ENCODED_PDF>`` to render the pdf. 

At this time I have a workaround by creating a new pdf (FPDF) with output set to 'D' (download) but this feels odd because the pdf is already created and downloading from a viewer seems as usual as printing the document.

**Comment by hrynko** [comment]
As was commented in #68, it seems to me that this does not quite fit into the concept of the library. But maybe we can get more opinions here.

**Reply by wesleyscherrenberg** [reply]
@hrynko i think this will be a nice feature, you could say that the print method is also not part of the concept

**Reply by hrynko** [reply]
Thank you for your comments, it is now available in `v1.2.0`
